examples/testbed_native_android/android/app/src/main/java/io/branch/testbed_native_android/MainActivity.java

```diff
@@ -78,7 +90,9 @@ protected void onDestroy() {
         if (mReactInstanceManager != null) {
             mReactInstanceManager.onHostDestroy(this);
         }
-    }
+
+        unsubscribeFromBranchLinks();
```

Comment:
This seems like we developer need  to be aware  of adding this code  on destroy of every activity. Can SDK some how intercept the lifecycle methods and call the  `subscribeToBranchLinks` method  and `unsubscribeFromBranchLinks `